### DISCUSSION TITLE
CLI for headless use

### DISCUSSION THREAD

**Discussion by theo-m** [lairdubois/lairdubois-opencutlist-sketchup-extension#417] [Ideas]
Hey I'm not a sketchup customer, I only use the webapp, and I'd love to use opencutlist without having to load it from the 300$/y software it's attached to.

Is there a way to run your software "headless" through a CLI with something that would look like this

```shell
opencutlist --options myoptions.yaml myfile.skp
```

I'd be happy to contribute, just not very familiar with ruby

**Comment by mobilarte** [comment]
Hi Theo,

This is currently not planned, because you can always use SketchUp 2017. I think you can download a model from the online version in 2017 format. Otherwise, there is an [extension ](https://github.com/Eneroth3/open-newer-version) that let's you open newer version from within 2017.

We do not know what the future plans for SketchUp are, therefore we would be hesitant to invest a lot of time into a feature like that.

**Reply by theo-m** [reply]
Great stuff, didn't know about sketchup 2017 - will look into it, thanks!
